recipe/meta.yaml

```diff
@@ -64,6 +64,12 @@ test:
     - jupyter lab build --dev-build=False --minimize=False
     - jupyter lab clean
 
+app:
+  entry: jupyter lab
+  icon: icon.png
```

Comment:
Most examples only show using a 64px or smaller, and square, icon. While the former is debateable, the latter is probably a good idea... I don't know how many places these actually get used.

Also, If we're going to start doing this across more projects, we _need_ to figure out some way to distinguish between Lab and Notebook and Xyz... This could be as simple as using the on-brand-font (oops, not open source, who could have anticipated this) or at least something _close_ and adding _Lab_ to the bottom, with a moon's worth of offset, as per the guidance. 

Even better, if this was available from the upstream sdist, in a well-known location, as that has a whole build chain that could handle updating and optimizing an icon.